### DISCUSSION TITLE
magnet clone ?

### DISCUSSION THREAD

**Discussion by NeoRoth** [rxhanson/Rectangle#673] [Q&A] [answered]
I accidentally found this app that looks a lot like a rectangle on the App Store:
[magnet] (https://apps.apple.com/fr/app/magnet/id441258766?mt=12)
I don't know if it's relevant but it generalizes me a lot that a company benefits from the work of the community.

Afterwards I am not in a position to say if this is really the case maybe it could attract someone here to overdo it.

**Comment by rxhanson** [comment, marked as answer]
Magnet was around long before Rectangle. Rectangle was originally based on Spectacle, but borrows some concepts from Magnet (which in turn borrowed concepts from Windows).

**Reply by NeoRoth** [reply]
Thank you, sorry, I got scared too quickly.
